infra/event-sync/src/main/java/com/simprints/infra/eventsync/event/remote/models/session/ApiEventScope.kt

```diff
@@ -22,6 +24,7 @@ internal data class ApiEventScope(
     val device: ApiDevice,
     val databaseInfo: ApiDatabaseInfo,
     val location: ApiLocation?,
+    @JsonInclude(Include.NON_EMPTY)
```

Comment:
I think you could add a test to ensure that you have what you want, and if people remove the annotation something will fail

Reply:
Good idea! Added a test.

Reply:
Good idea! Added a test.